### ISSUE TITLE
Feature Request: Log filtering

### ISSUE THREAD

**Issue by theothertomelliott** [windmilleng/tilt#3033]
We're getting more of our engineers using Tilt and an early piece of feedback was that it would be nice to be able to filter out log messages.

We have a few spammy messages from third party packages and being able to omit them through a regex or similar would be very powerful.

**Comment by nicks** [comment]
Interesting! Are these build logs (like warnings from third-party packages) or runtime logs (like some debug noise from a library you're calling into)?

**Comment by theothertomelliott** [comment]
The example I was sent was runtime logs from a library.

**Comment by maets** [comment]
Yes! Currently testing stuff locally and wishing the "all resources" log view could function as a "helicopter perspective microservice overview/debugging tool".

But due to some really chatty 3rd party resources this flow is practically impossible to follow. 

I imagine like a checkbox per resource to show in the log view - topped with a select all/none checkbox - would be a really nice feature for my case.
